.github/workflows/build.yml

```diff
@@ -4,18 +4,14 @@ on: ["pull_request", "push"]
 
 jobs:
   build:
-    strategy:
-      matrix:
-        java: ["16"]
-        os: ["ubuntu-latest"]
-    runs-on: "${{ matrix.os }}"
+    runs-on: "ubuntu-latest"
     steps:
       - name: "Checkout Repository"
         uses: "actions/checkout@v2.3.4"
-      - name: "Setup JDK ${{ matrix.java }}"
+      - name: "Setup JDK 11"
         uses: "actions/setup-java@v2"
         with:
           distribution: "adopt"
-          java-version: "${{ matrix.java }}"
+          java-version: "11"
```

Comment:
This step can use 16 like outlined in my previous comment.
It sets the Java version of the GitHub action runner, but has no impact on the Java version of the project.